src/coreclr/jit/compiler.cpp

```diff
@@ -3491,14 +3498,43 @@ bool Compiler::compStressCompile(compStressArea stressArea, unsigned weight)
     {
         if (verbose)
         {
-            printf("\n\n*** JitStress: %ws ***\n\n", s_compStressModeNames[stressArea]);
+            printf("\n\n*** JitStress: %s ***\n\n", s_compStressModeNames[stressArea]);
```

Comment:
This also fixes this printf for PAL (it was just printing as "JitStress: %ws" on non-Windows builds)